editions/server-external-js/tiddlers/external/tiddlywiki.files

```diff
@@ -0,0 +1,8 @@
+{
+	"tiddlers": [
+		{
+			"file": "../../../tw5.com/tiddlers/webserver/Using the external JavaScript template.tid",
```

Comment:
I know we do things like this already, but this does strike me as super brittle. In particular, it won't be clear to anyone subsequently performing maintenance on that docs file that it is pulled in here. Perhaps add a comment to the top of the docs file specifically calling out that it is included elsewhere.

Reply:
Should I add some info as html comment eg: `<!-- this tiddler is imported into the server-external-js edtion -->` .. Or should I use a visible text

Reply:
I was thinking of a comment; the concern is for people editing that tiddler in the future:

```
<!--
This tiddler is also included in the "server-external-js" edition. Take care before editing or moving it.
-->
```